libafl_libfuzzer/libafl_libfuzzer_runtime/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "libafl_libfuzzer_runtime"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+default = ["mimalloc", "fork"]
+fork = ["libafl/fork"] # Enables forking mode for the LibAFL launcher (instead of starting new processes)
```

Comment:
@addisoncrump FYI, removing default_features should now use process restarting instead of forks (if this uses Launcher).
We should probably test this at some point :D